src/attributes/attr.js

```diff
@@ -7,7 +7,14 @@ define( [
 ], function( jQuery, access, support, rnotwhite ) {
 
 var boolHook,
-	attrHandle = jQuery.expr.attrHandle;
+	attrHandle = jQuery.expr.attrHandle,
+
+	// Exclusively lowercase A-Z in attribute names (gh-2730)
+	// https://dom.spec.whatwg.org/#converted-to-ascii-lowercase
+	raz = /[A-Z]/g,
```

Comment:
Not that it makes a lot of difference, but you could use `[A-Z]+` here to reduce the potential number of function calls?